subprojects/docs/src/docs/userguide/logging.adoc

```diff
@@ -118,23 +113,21 @@ Internally, Gradle uses Ant and Ivy. Both have their own logging system. Gradle
 
 There are many tools out there which still use standard output for logging. By default, Gradle redirects standard output to the `QUIET` log level and standard error to the `ERROR` level. This behavior is configurable. The project object provides a link:{javadocPath}/org/gradle/api/logging/LoggingManager.html[LoggingManager], which allows you to change the log levels that standard out or error are redirected to when your build script is evaluated.
 
-=== Example: Configuring standard output capture
+.Configuring standard output capture
+====
+include::sample[dir="userguide/tutorial/logging",files="build.gradle[tags=capture-stdout]"]
 
-[source.multi-language-sample,groovy]
-.build.gradle
-----
-include::{samplesPath}/userguide/tutorial/logging/build.gradle[tag=capture-stdout]
-----
+include::sample[dir="userguide/tutorial/loggingKts",files="build.gradle.kts[tags=capture-stdout]"]
```

Comment:
can we do something with this extension so that we enforce that kotlin and groovy samples are named consistently?  I think it would also be OK if we arranged the samples more in a hierarchy (`userguide/tutorial/logging/kotlin` and `userguide/tutorial/logging/groovy`).